### ISSUE TITLE
[Feature Request]: Landing Page Design 

### ISSUE THREAD

**Issue by dev129** [Soujanya2004/wanderlust-2024#343]
Hi @Soujanya2004 
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Hi @Soujanya2004 
I noticed that when a user lands upon your website it directly shows them the listing , I want to redesign the Landing Pge making it more sttarctive to user and not fording them to sign up instead they would do it themselves.

### Use Case

Suppose I thought of staying in Karnataka , so forstly when I land on your site I must see a beautiful landing page , which will make sure that I should book from your site only , there I shall implement my check in and checkout dates and the location and then get the data of only those places which are in Karnataka. and for that I must then Login , I should not be forced to login to see places.

### Benefits

This will ensure trust among customers, make them feel good, and also get them places which they want to see raher them showing them places from japan, south korea etc. when they want to see houses in India.

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

**Comment by Soujanya2004** [comment]
@dev129 the booking functionality is not implemented ryt now
you can work on the landing page

**Comment by dev129** [comment]
Hye @Soujanya2004 , a small question are you sure that the documentation is right since i am getting this error 
![image](https://github.com/user-attachments/assets/016862ff-99b9-4ce6-91ad-01cb5f43e3c5)
Please try running on your system and let me know if it works for you or not

**Comment by Soujanya2004** [comment]
@dev129 npm install
node app.js

**Comment by Soujanya2004** [comment]
follow documentation its proper

**Comment by dev129** [comment]
Hi @Soujanya2004 , I have been trying to make things work our a bit but this is the proble =m i am facing 
![image](https://github.com/user-attachments/assets/f0d99449-943c-49c2-9da0-64cfc545e2f2)
also when i add .env and node_modules/ igit igonre file and savre it , it is still not saving it , please letme now what to do ?

**Comment by Soujanya2004** [comment]
SECRET=mysecret @dev129

**Comment by dev129** [comment]
Hi @Soujanya2004 this might be frustrating for you as much as it is for me 
![image](https://github.com/user-attachments/assets/1998c5ff-a2b6-4ba9-9d03-185a481eb188)
and is this the normal landing page during developement 
![image](https://github.com/user-attachments/assets/5c477ae9-dace-49b8-a7f9-fa37602adbf8)

**Comment by Soujanya2004** [comment]
@dev129 ATLAS_DB_TOKEN=mongodb://127.0.0.1:27017/wanderlust
add this for ur atlas token

**Comment by Soujanya2004** [comment]
> Hi @Soujanya2004 this might be frustrating for you as much as it is for me the below image is for /listing ![image](https://private-user-images.githubusercontent.com/76431221/382548652-1998c5ff-a2b6-4ba9-9d03-185a481eb188.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2MjAxNTIsIm5iZiI6MTczMDYxOTg1MiwicGF0aCI6Ii83NjQzMTIyMS8zODI1NDg2NTItMTk5OGM1ZmYtYTJiNi00YmE5LTlkMDMtMTg1YTQ4MWViMTg4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDA3NDQxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM2ODYyZDI3ZTE5MmJjNWFiY2VjZWQ0ZmVmOWE2OGI1NDMzZTViYjI4ODBjMWU5YzM5ZjI0ODQ1ZjIyOWFjYmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.3ReXZKi9tGCJMGDyiLIE3MExZHGoXlEu1wHWwzUCsog) and is this the normal landing page during developement ![image](https://private-user-images.githubusercontent.com/76431221/382548665-5c477ae9-dace-49b8-a7f9-fa37602adbf8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2MjAxNTIsIm5iZiI6MTczMDYxOTg1MiwicGF0aCI6Ii83NjQzMTIyMS8zODI1NDg2NjUtNWM0NzdhZTktZGFjZS00OWI4LWE3ZjktZmEzNzYwMmFkYmY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDA3NDQxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4OWU1M2MxMGMxNzU5MDRmMTk5YTE0MTM0ZmNkNDJlNjMwYjVlNzBmYjgzMzM1M2IzOGNlZDllYjQzMjk5NTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.zS4JhM3kMumjtX_7y4dC6WxcHOB5g7RysQ5uOGxdcQ4) A very interesting thing is that after sometime the main page also becomes like the first images, its hilarious

oh god, the first image error has something to do with ur mongodb only
i faced same issue and fixed it now

**Comment by Soujanya2004** [comment]
@dev129 if u get that error solved, your landing page is almost set up just need to remove that error message displayed and add some interesting landing page section

**Comment by dev129** [comment]
https://github.com/user-attachments/assets/6c51dfa9-d33a-4c84-bdd2-6c0d05707438

**Comment by dev129** [comment]
Hi @Soujanya2004 , are you there ?

**Comment by Soujanya2004** [comment]
@dev129 the recording u have uploaded is improper plz check

**Comment by dev129** [comment]
@Soujanya2004 , hols on right now , its hard to contact you.

**Comment by Soujanya2004** [comment]
@dev129 i am really sorry for that, my exams are going on so little inactive

**Comment by dev129** [comment]
@Soujanya2004 , same pinch , anyways here are the updated version after this if you find anything wrong , let me know all of them once, and I shall then message you only once at a time , in that way you just have to check , merge it , no wastage of your time.
![image](https://github.com/user-attachments/assets/195460d3-c506-492c-be9c-97da6f782ac2)
Also all the best for your exam

**Comment by Soujanya2004** [comment]
@dev129 looks gud to me, and it is displayed in / route ryt
thanks, ace ur exams btw😄

**Comment by dev129** [comment]
Yes this is the homepage design , so is it a yes ?

**Comment by Soujanya2004** [comment]
@dev129 yes perfect, i hope you have included navbar into it

**Comment by dev129** [comment]
@Soujanya2004 , navbar would have made the look of landing page dull and as for footer :
![image](https://github.com/user-attachments/assets/20cc43fb-2e24-4737-9658-5d1c1467302b)

**Comment by Soujanya2004** [comment]
@dev129 please use the same footer boilerplate 
and if there is no navbar how will user navigate through website

**Comment by Soujanya2004** [comment]
> @dev129 please use the same footer boilerplate and if there is no navbar how will user navigate through website

you can raise another issue and work on that no issues, i will assign u
